### PR TITLE
SOF-7459: platform access video + other improvements

### PR DESCRIPTION
**Pull request by pranabdas**
Preview: https://deploy-preview-279--mat3ra-documentation.netlify.app/tutorials/overview/

### CODE REVIEW

File: lang/en/docs/tutorials/platform-access.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    "youTubeCaptions": [
+        {
+            "text": "Hi, <break time='0.25'/> welcome to matera tutorial series once again. In this short video, we will give a glimpse of various ways of accessing our platform.",
+            "startTime": "00:00:00.000",
+            "endTime": "00:00:09.000"
+        },
+        {
+            "text": "There are three main ways to access our simulation platform, namely, <break time='0.25'/> the web interface using a browser, <break time='0.5'/> command line interface via S S H, <break time='0.25'/> and A P I access. These different modes of access are designed keeping in mind various use cases. For example, <break time='0.25'/> a novice user will probably find the web browser interface most user friendly to run his or her D F T simulations, <break time='0.25'/> while an enterprise user might find our A P I access is suitable to integrate our services with their existing client application. However, these different modes of access are complimentary to each other, and oftentimes, a user might want to use more than one mode or all three modes to interact with our platform depending on the task at hand.",
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/platform-access.json:21]: Screenshot 2024-10-10 at 3 12 29 PM
When we use long text snippets like this, it's hard to follow the video with captions. Could we split?

File: lang/en/docs/tutorials/platform-access-2.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            "endTime": "00:00:43.000"
+        },
+        {
+            "text": "That's all for this short platform introduction. Now, it's your turn to explore more. Thank you for following along. <break time='0.1'/>",
+            "startTime": "00:00:45.000",
+            "endTime": "00:00:52.000"
+        }
+    ],
+    "youTubeId": ""
+}
```
Comment (timurbazhirov) [inline lang/en/docs/tutorials/platform-access-2.json:32]: Why are there 2 files?

File: lang/en/docs/tutorials/platform-access.json (hunk 1 above)
Comment (pranabdas) [reply lang/en/docs/tutorials/platform-access.json:21]: For the current video, I have regenerated the subtitles using YouTube's automatic subtitle feature. In the future, we should have smaller text segments.

File: lang/en/docs/tutorials/platform-access-2.json (hunk 2 above)
Comment (pranabdas) [reply lang/en/docs/tutorials/platform-access-2.json:32]: We are exceeding the limit of how much text-to-speech can be synthesized in one call. Perhaps, we can use the [LongAudio](https://cloud.google.com/python/docs/reference/texttospeech/latest/google.cloud.texttospeech_v1.services.text_to_speech_long_audio_synthesize.TextToSpeechLongAudioSynthesizeClient) client, but in this case, the response does not include actual audio, instead, it requires setting up a GCP bucket. So, we will need additional steps to get the audio files locally.

File: lang/en/docs/tutorials/overview.md [hunk 3]
```diff
@@ -1,3 +1,73 @@
...
+Below we present a short video on various ways of accessing Mat3ra platform.
+The sub-pages under this section contain detailed tutorials. You can find
+specific tutorial from the index below or locate the corresponding entry in the
+sidebar navigation.
+
+<div class="video-wrapper">
+<iframe class="gifffer" width="100%" height="100%" src="https://www.youtube.com/embed/vERldqp77bU" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+## Index of Tutorials
```
[the remark this answers is not in the document]
Comment (pranabdas) [reply lang/en/docs/tutorials/overview.md:12]: I have written a script to generate the index.

**Review by timurbazhirov — APPROVED** [review]
